workers/unity/Packages/com.improbable.gdk.transformsynchronization/Tests/Editmode/Systems/TickSystemTests.cs

```diff
@@ -10,20 +10,21 @@ public class TickSystemTests
         [Test]
         public void GlobalTick_gets_incremented_in_OnUpdate()
         {
-            using (var world = new World("test-world"))
-            {
-                var tickSystem = world.GetOrCreateManager<TickSystem>();
-
-                Assert.AreEqual(0, tickSystem.GlobalTick);
-
-                tickSystem.Update();
-
-                Assert.AreEqual(1, tickSystem.GlobalTick);
-
-                tickSystem.Update();
-
-                Assert.AreEqual(2, tickSystem.GlobalTick);
-            }
+            // using (var world = new World("test-world"))
+            // {
+            //     var tickSystem = world.GetOrCreateManager<TickSystem>();
+            //
+            //     Assert.AreEqual(0, tickSystem.CurrentPhysicsTick);
+            //
+            //     tickSystem.Update();
+            //
+            //     Assert.AreEqual(1, tickSystem.CurrentPhysicsTick);
+            //
+            //     tickSystem.Update();
+            //
+            //     Assert.AreEqual(2, tickSystem.CurrentPhysicsTick);
+            // }
+            Assert.IsTrue(false);
```

Comment:
This will never pass CI - remove the test if we aren't going to rewrite before merging this PR and open a ticket to rewrite the test

Reply:
Ah yes. Forgot about that. This was there for exactly that purpose :P